submodules/eks/variables.tf

```diff
@@ -144,3 +144,8 @@ variable "node_groups_kms_key" {
   description = "if set, use specified key for the EKS node groups"
   default     = null
 }
+
+variable "eks_custom_role_maps" {
+  type        = list(object({ rolearn = string, username = string, groups = list(string) }))
+  description = "Custom role maps for aws auth configmap"
```

Comment:
@Secretions Since this is a "custom..." item, I think we should add a `null` default. If there's a more sensible default, I'm fine with that too (since I see there's no `if` in `submodules/k8s/templates/aws-auth.yaml.tftpl`).

Reply:
I copied the default from the root folder's variables to here, which is `[]`. Since it loops through, it seems like the sensible default here. I'm honestly not sure why the default needs to be in all three places, though. Since both the `eks` and `k8s` submodules aren't intended to be used separately, they should always get the input from the root `/variables.tf` and inherit its default. On the other hand, if we change a default, we now have to remember to change it in three places (not that I anticipate ever changing it once this merges).